src/content/developers/tutorials/send_token_etherjs/index.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Sending Tokens Using ethers.js
+description: Beginner friendly guide to sending tokens using ethers.js.
+author: Kim YongJun
+tags: ["ETHERS.JS", "ERC-20", "TOKENS"]
+skill: beginner
+lang: en
+sidebar: true
+published: 2021-04-06
+---
+
+
+
+## Send Token Using ethers.js(5.0) {#send-token}
+
+### In This Tutorial You'll Learn How To {#you-learn-about}
+
+* Import ethers.js
+* Transfer token
+* Set gas price according to the network traffic situation
+
+### To-Get-Started {#to-get-started}
+
+To get started, we must first import the ethers.js library into our javascript 
+Include ethers.js(5.0)
+
+### Installing {#install-ethersjs}
+
+```shell
+/home/ricmoo> npm install --save ethers
+```
+
+ES6 in the Browser
+
+```html
+<script type="module">
+    import { ethers } from "https://cdn.ethers.io/lib/ethers-5.0.esm.min.js";
+    // Your code here...
+</script>
+```
+
+ES3(UMD) in the Browser
+
+```html
+<script src="https://cdn.ethers.io/lib/ethers-5.0.umd.min.js"
+        type="application/javascript"></script>
+```
+
+### Parameters {#param}
+
+1. **`contract_address`**: Token contract address (contract address is needed when the token you want to transfer is not ether)
+2. **`send_token_amount`**: The amount you want to send to the receiver
+3. **`to_address`**: The receiver's address
+4. **`send_account`**: The sender's address
+5. **`private_key`**: Private key of the sender to sign the transaction and actually transfer the tokens
+
+## Notice {#notice}
+
+`signTransaction(tx)` is removed because `sendTransaction()` does it internally.
+
+## Sending Procedures {#procedure}
+
+### 1. Connect to network (testnet) {#connect-to-network}
+
+#### Set Provider (Infura) {#set-provider}
+
+Connect to Ropsten testnet
+
+```javascript
+window.provider = new InfuraProvider("ropsten");
+```
+
+### 2. Create wallet {#create-wallet}
+
+```javascript
+let wallet = new ethers.Wallet(private_key);
+```
+
+### 3. Connect Wallet to net {#connect-wallet-to-net}
+
+```javascript
+let walletSigner = wallet.connect(window.ethersProvider);
+```
+
+### 4. Get current gas price {#get-gas}
+
+```javascript
+window.ethersProvider.getGasPrice(); // gasPrice
+```
+
+### 5. Define Transaction {#define-transaction}
+
+These variables defined below are dependent on `send_token()`
+
+### Transaction parameters {#transaction-params}
+
+1. **`send_account`**: address of the token sender
+2. **`to_address`**: address of the token receiver
+3. **`send_token_amount`**: the amount of tokens to send
+4. **`gas_limit`**: gas limit
+5. **`gas_price`**: gas price
+
+[see below for how to use] (#how-to-use)
```

Comment:
No space =), and lets capitalize the start
```
[See below for how to use](#how-to-use)
```
[See below for how to use](#how-to-use)